docs/mappings.md

```diff
@@ -42,34 +42,66 @@ As a general standard, use American English spelling for words. For example, pre
 1. **Names should be named based on the parameter types and context of use.**
     - They should be verbose and use complete words. Avoid omitting essential information for the purpose of keeping the 
       name compact.
+    - Use the surrounding mojang class, field, and method names to infer parameter names. Prefer matching the mojang naming
+      rather than trying to improve on it.
     - {:.small-text} _Examples:_ `BlockPos adjacentPos, BlockPos currentPos`, not `BlockPos pos1, BlockPos pos2`
 
 1. **Avoid abbreivations or acronyms.**
-    - _Exception:_ Common or well-known abbreviations can be used: `IO` (input-output), `Id` (identifier)
+    - _Exception:_ Common or well-known abbreviations can be used: `IO` (input-output), `Id` (identifier).
     - _Exception:_ Names for common classes can be shortened: `BlockPos pos`, `BlockState state`, `WorldGenLevel level`
 
-> **Note about lambda parameters:**
-> 
-> Currently, there is no fullly automated solution for verifying that mapped lambda parameters do not conflict with 
-> parameters (both method and other lambdas) or local variables. Care should be taken when naming and reviewing these.
-> 
-> As a workaround, Parchment has a 'checked' export which, among other things, strips out names for lambda parameters,
-> so projects which require non-conflicting parameter names (such as for recompilation of decompiled source) may use 
-> that instead.
-
 ## Javadocs
 
-When documenting, try to keep explanations simple and concise without sacrificing accuracy. Avoid use of overly complex
-or lengthy words where simple and short words are sufficient.
-
-Documenting members should be prioritized into the following: public members, then important members, then non-obvious 
-members (commonly, getters which return numbers or opaque objects, such as NBT tags.) Methods should be documented over
-their corresponding fields.
-
 {:.list-separation}
 
-1. **Use fully qualified class names in javadoc tags which link to classes or their members.**
-    - Some of these javadoc tags are `{@link}` and `@see`.
+1. **Use complete, meaningful, concise sentences**
+    - Try and keep explanations simple and concise without sacrificing accuracy. Avoid the use of overly complicated 
+      words where short and simple descriptions are sufficient.
+    - Avoid the use of domain specific knowledge that the reader would not be assumed to have if it is not necessary.
```

Comment:
This should probably be clarified in a way that is easier to read. How does someone determine if domain specific knowledge is "necessary"?